backend/pkg/messages/legacy-message-transform.go

```diff
@@ -1,14 +1,6 @@
 package messages
 
 func transformDeprecated(msg Message) Message {
```

Comment:
I think in this case we can completely remove this method

Reply:
I suspect we might need it soon again - (any time when we extend some message)